genie-agent/src/main/java/com/netflix/genie/agent/GenieAgentApplication.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.agent;
```

Comment:
I think the general recommendation is to keep the main classes in the root directory above most of your other packages. So `com.netflix.genie` not `agent`. This helps with classpath scanning of your packages especially if you pull things in from something like `com.netflix.genie.common` or something else down the road.